### ISSUE TITLE
Alert - Ensure images provide informative alternative text - (2036569776)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5561]
### Module: 
 07a Alert 

### Violation: 
 Ensure images provide informative alternative text 

![image](https://user-images.githubusercontent.com/5023024/202814656-fb0d7980-39ca-44d2-a706-28b97e70c345.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036569776 

### Severity: 
 10 

### Description: 
 [Issue]
There are meaningful svg images without textual equivalents. Examples include:
- Warring icons inside the alerts.

[User Impact]
Screen reader users will not receive an accurate impression of the contents of these images.

[Code Reference] 
 ```
<calcite-icon aria-hidden="true" icon="exclamation-mark-triangle" scale="s" calcite-hydrated=""></calcite-icon>
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg">
....
</svg>

 ``` 

### Note: 
 [Suggestion]
Ensure images provide informative alternative text. Textual equivalents must be both concise and descriptive. In this case remove aria-hidden="true" from the ``` <Calcite> ``` element. In the case of svg elements, it is recommended to add role="img", ``` <title> ``` with the text equivalent and aria-labelledby set to the id of the title element

[Compliant Code Example] 
 ```
<calcite-icon  icon="exclamation-mark-triangle" scale="s" calcite-hydrated=""></calcite-icon>
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="title">
<title id="title">medium speed Warring</title>
....
</svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Similar to the need for https://github.com/Esri/calcite-components/issues/5598

**Comment by geospatialem** [comment]
Mitigated in a PR prior to the 1.0.5 release, where the svg uses `aria-hidden="true"`.
